src/components/AnalysisModal.js

```diff
@@ -187,11 +190,11 @@ export const AnalysisModal = withDisplayName('AnalysisModal')(
     }, [
       renderToolButtons(),
       h(Dropzone, {
-        accept: `.${tools.Jupyter.ext}, .${tools.RStudio.ext}`,
+        accept: `.${tools.Jupyter.ext.join(', .')}, .${tools.RStudio.ext.join(', .')}`,
         style: { flexGrow: 1, backgroundColor: colors.light(), height: '100%' },
         activeStyle: { backgroundColor: colors.accent(0.2), cursor: 'copy' },
         onDropRejected: () => reportError('Not a valid analysis file',
-          'The selected file is not a .ipynb notebook file or an .Rmd rstudio file. Ensure your file has the proper extension.'),
+          'The selected file is not one of the supported types: .ipynb, .R, .Rmd. Ensure your file has the proper extension.'),
```

Comment:
could we generate the string message from the tools object `ext` arrays via join?

Reply:
Yeah, that's an easy change so will do. Thanks for the suggestion!